docs/framework/guides/deep-dive/observables.md

```diff
@@ -0,0 +1,303 @@
+---
+category: framework-deep-dive
+---
+
+# Observables
+
+{@link module:utils/observablemixin~Observable Observables} are objects which have properties that can be observed. That means when the value of such property changes, an event is fired by the observable and the change can be reflected in other pieces of the code that listen to that event.
+
+Any class can become observable; all you need to do is mix the {@link module:utils/observablemixin~ObservableMixin} into it:
+
+```js
+import ObservableMixin from '@ckeditor/ckeditor5-utils/src/observablemixin';
+import mix from '@ckeditor/ckeditor5-utils/src/mix';
+
+export default class AnyClass {
+	// ...
+}
+
+mix( AnyClass, ObservableMixin );
+```
+
+Observables are useful when it comes to managing the state of the application, which can be dynamic and, more often than not, centralized and shared between components of the application. One observable can also propagate its state (or its part) to another using [property bindings](#property-bindings).
+
+Observables can also [decorate their methods](#decorating-object-methods) which makes it possible to control their execution using event listeners, giving external code some control over their behavior.
+
+## Making properties observable
+
+Having mixed the {@link module:utils/observablemixin~ObservableMixin} into your class, you can define observable properties. To do that, use the {@link module:utils/observablemixin~ObservableMixin#set `set()` method}. Let's set a couple of properties and see what they look like in a simple `Command` class:
+
+```js
+export default class Command {
```

Comment:
This is confusing. Especially the `new Command( 'thing' )` used later on. We should base the docs on some realistic code and we don't write a code like this.